metagov/metagov/core/views.py

```diff
@@ -274,13 +274,15 @@ def plugin_auth_callback(request, plugin_name):
     plugin_views = importlib.import_module(f"metagov.plugins.{plugin_name}.views")
 
     try:
-        return plugin_views.auth_callback(
+        response = plugin_views.auth_callback(
             type=type,
             code=code,
             redirect_uri=redirect_uri,
             community=community,
             state=state_to_pass,
+            request=request
```

Comment:
I think this line needs to be updated with `request=None` so we don't get an unexpected argument error

https://github.com/metagov/metagov-prototype/blob/bef16576044e9aa56cedfe825ab6aeeb478ac9bc/metagov/metagov/plugins/slack/views.py#L69

Reply:
Good catch. I can fix that, although alternatively we may want to put ``**kwargs`` in the function signature so that we gracefully catch unexpected parameters and don't need to update every time we add a parameter.

Reply:
Ok, updated slack's `auth_callback` signature to have *args and **kwargs. So we can add more parameters in the call from core without worrying about breaking the signature in the future. We can recommend in the docs that all plugin implementations of auth_callback do this.